odoo/tools/mail.py

```diff
@@ -282,6 +296,33 @@ def is_html_empty(html_content):
     tag_re = re.compile(r'\<\s*\/?(?:p|div|span|br|b|i)(?:(?=\s+\w*)[^/>]*|\s*)/?\s*\>')
     return not bool(re.sub(tag_re, '', html_content).strip())
 
+def bracket_to_qweb_instructions(text):
```

Comment:
Hello,

Why should we support a "jinja-like" writing ? Anyway we are breaking all templates. Has it any use to keep another way of writing templates ? Why don't we simply use directly ``<t />`` tags everywhere and simplify everything ?

As smart people say, every regex-based parsing will fail one time or another. I think this is keeping an unnecessary layer. What is the rationale behind this code ?

Reply:
Same answer as here: https://github.com/odoo/odoo/pull/70045/files#r655182099

The parsing is quite minimal. If it breaks, it would not be that difficult to add a test and fix it.